bx_py_utils/stack_info.py

```diff
@@ -0,0 +1,32 @@
+import inspect
+from pathlib import Path
+from typing import Union
+
+from bx_py_utils.path import assert_is_file
+
+
+def last_frame_by_path(file_path: Union[Path, str]):
```

Comment:
type annotation here

Reply:
not needed, because now it raised `FrameNotFound` and not just `None` ; )